Source/Blazorise/Base/BaseComponent.cs

```diff
@@ -142,6 +143,22 @@ protected override async Task OnAfterRenderAsync( bool firstRender )
         protected virtual Task OnFirstAfterRenderAsync()
             => Task.CompletedTask;
 
+        /// <inheritdoc/>
+        protected override void Dispose( bool disposing )
+        {
+            ClassBuilder = null;

```

Comment:
Are there really any benefits in setting the member instances to null? According to this https://blog.stephencleary.com/2010/02/q-should-i-set-variables-to-null-to.html, and other sources I'm reading, it should be avoided.

Reply:
I haven't read the blog yet. But generally you don't need to do it, yes.
But in thsi case. Since we have the problem that the components are being kept in memory it won't release these At least from my tests... See prints...

Reply:
Just wanted to confirm if you're aware of it. I will merge.